src/main/kotlin/DetektDemo.kt

```diff
@@ -0,0 +1,5 @@
+class DetektDemo {
+    fun demo1() {
+        val x = 5
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/57411e4d0d97cbc88375c5ebcd69da75edbd6feb/src/main/kotlin/DetektDemo.kt#L3"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>